packages/cli/lib/project.js

```diff
@@ -63,6 +63,29 @@ class Project extends EventEmitter {
     this.verbose = verbose;
   }
 
+  /**
+  * Runs pandoc command for buildEpub and buildMobi
+  * @param {string} the destination for the output file
+  * @param {string} the path to the cover image
+  */
+  runPandoc(outputFile, cover) {
+    const args = [
+      `-f`,
+      `html-native_divs+native_spans`,
+      `-t`,
+      `epub`,
+      path.join("html","epub.xhtml"),
+      `-o`,
+      outputFile,
+      `--epub-metadata=${path.join("html", "dc.xml")}`,
+      cover,
+      `--template=${path.join("html", "template.xhtml")}`,
+      `--css=${path.join(this.config.publishDir, "css", "epub.css")}`,
+      `-s`
+    ];
```

Comment:
Using unabbreviated command flag-name=value makes this more readable and is more explicit, documents, the command being run.
```suggestion
    // @see https://pandoc.org/MANUAL.html#general-options
    const args = [
      '--from=html-native_divs+native_spans',
      `--to=epub ${path.join("html", "epub.xhtml")}`,
      `--output=${outputFile}`,
      `--epub-metadata=${path.join("html", "dc.xml")}`,
      cover,
      `--template=${path.join("html", "template.xhtml")}`,
      `--css=${path.join(this.config.publishDir, "css", "epub.css")}`,
      '--standalone'
    ];
```

Reply:
Great suggestion, this is way more readable